index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Ethiopian Food</title>
+		<link rel="stylesheet" type="text/css" href="quiz.css">
+	</head>
+	<body>
+		<div class="flex">
+			<article class="restaurantsCard">
+				<header class="dish">
+					<h1>Dish1</h1>
+				</header>
+				<section>
+					<img src="img/food1.png" alt="food1.png" width="250px" height="250px">
```

Comment:
instead of inline styling for height and width can you add a class to the image?  then set the height/width to a percentage